app/Models/DuesTransactionMerchandise.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\Pivot;
+
+class DuesTransactionMerchandise extends Pivot
+{
+    /**
+     * The attributes that should be mutated to dates.
+     *
+     * @var array<string>
+     */
+    protected $casts = [
+        'provided_at' => 'datetime',
+    ];
+
+    /**
+     * Indicates if the IDs are auto-incrementing.
+     *
+     * @var bool
+     */
+    public $incrementing = true;
+
+    public function providedBy(): BelongsTo
+    {
+        return $this->belongsTo(User::class, 'provided_by');
+    }
+
+    /**
+     * Returns the name field of the User in providedBy. This exists because Nova was not cooperating.
+     */
+    public function getProvidedByNameAttribute(): ?string
+    {
+        return optional($this->providedBy)->name;
+    }
```

Comment:
This kills the link I assume? Can we add a tech debt item to try and fix this at some point?

Reply:
Yes, it does. I tried several different ways to get it to use the relationship on the pivot model to no avail.